### ISSUE TITLE
Duchenne/Becker Muscular Dystrophy and splicing

### ISSUE THREAD

**Issue by SiminaB** [greenelab/deep-review#565]
I think the discussion of Duchenne/Becker MD and splicing should be a little clearer and also include specific references. In particular, Duchenne is the more severe form and Becker the less severe form of the same kind of disorder (but in reality it can be more of a continuum - see last paragraph here), both of them being caused by mutations to the same gene (dystrophin) - the mutations in Becker are usually "in frame" versus the ones in Duchenne which usually cause frameshifts (or are nonsense mutations) and lead to no functional protein being made.

I'm not familiar with Duchenne and Becker being classified as "splicing-related diseases" per se. The splicing *I think* comes in more at the therapy development stage in this case. For example, one class of therapies being considered are "exon skipping therapies" (one recently got conditionally approved by the FDA) which skip the mutated exon to get back in frame. The idea is to try to convert the more severe Duchenne phenotype to be closer to a less severe Becker phenotype. Another class of therapies being considered that is related to splicing is to use gene-therapy with a smaller version of dystrophin ("micro-dystrophin"). At 79 exons, dystrophin is the largest known human gene, so it is difficult to deliver the entire gene, which is why researchers are working on delivering a smaller version that still has the most important domains.
Here is one review on exon skipping (there a bunch): http://www.sciencedirect.com/science/article/pii/S0169409X15001015 
And a review on genetic approaches, including mini-dystrophin:
https://www.nature.com/nrg/journal/v14/n6/full/nrg3460.html

It could be that there are splicing issues for some specific types of Duchenne/Becker as well, as the disease course can still be quite heterogeneous despite it being a single-gene disorder. While there is overall a strong correlation between the mutation disrupting the reading frame and the phenotype, there are cases which are "genetically Becker but clinically Duchenne" as well as vice-versa and intermediate phenotypes. However, this should be stated clearly, for instance, in the example with the skipping of exon 31 in Becker.

**Comment by cgreene** [comment]
Pinging @agitter - do you recall who addressed that section? It'd be nice to bring them in here. Subsequent to that discussion (or before), @SiminaB if you'd like to edit the edit the text please file a pull request making the changes you'd like to see/agree on so that you can receive credit for the contribution.

**Comment by agitter** [comment]
I believe @bdo311 wrote that section.  Is that correct @bdo311?

If the Duchenne/Becker example is complex, should we switch it to a more obvious splicing-related disease?  The discussion above is extremely informative, but I don't think we want to go into that much detail in the splicing sub-section here.

**Comment by bdo311** [comment]
Thanks for contributing this info @SiminaB. I agree that Duchenne is more complicated and probably shouldn't be described as a "mis-splicing disease". Based on http://www.nature.com/nrg/journal/v17/n1/fig_tab/nrg.2015.3_F1.html we could rewrite this as "We already understand how mis-splicing of a single gene can cause disease. For instance, splicing mutations in the lamin A (LMNA) gene can lead to specific variants of dilated cardiomyopathy and limb girdle muscular dystrophy."

**Comment by agitter** [comment]
I'm in favor of a more straightforward example like what @bdo311 suggested.  Shall we proceed with a pull request in this direction?

**Comment by SiminaB** [comment]
That sounds good to me!

**Comment by SiminaB** [comment]
By the way, I'd be happy to open a pull request concerning this, just didn't want to have more than one open at the same time.

**Comment by bdo311** [comment]
Go for it @SiminaB! Sorry I haven't been super active here lately...

**Comment by agitter** [comment]
@SiminaB Please feel welcome to open multiple pull requests.  Specifically, it would be great if you can help close this issue with a small pull request.  The only thing I'd like to avoid is having multiple open pull requests that modify the same section because then we'll have to merge manually.

For pull requests like this one that only touch a sentence or two, I'll try to review and merge quickly.  Those that change the tone or content, like #566, take more time and discussion to merge.  I admittedly haven't been keeping up with those, but I like to see them and will get to them.

**Comment by SiminaB** [comment]
OK, done! Turns out there is more alternative splicing than I knew about for Duchenne/Becker as well (e.g. 5% of mutations for DMD may be splice-site disrupting) but the example that was given before for Becker actually seems to be for a case that converts the more severe Duchenne phenotype into a less severe intermediate one, so it was more disease-ameliorating than disease-causing. Anyway, I think it's clearer now with @bdo311's text. I also added a sentence on therapies, using the same reference.
